Mathlib/Algebra/Algebra/Unitization.lean

```diff
@@ -773,8 +773,11 @@ def starLift : (A →⋆ₙₐ[R] C) ≃ (Unitization R A →⋆ₐ[R] C) :=
       induction x using Unitization.ind
       simp [map_star] }
   invFun := fun φ ↦ φ.toNonUnitalStarAlgHom.comp (inrNonUnitalStarAlgHom R A),
-  left_inv := fun φ => by ext; simp,
-  right_inv := fun φ => Unitization.algHom_ext'' <| by simp }
+  left_inv := fun φ => by -- ext; simp,
+    ext
+    simp
+  right_inv := fun φ => Unitization.algHom_ext'' <| by
+    simp }
```

Comment:
These changes don't seem necessary

Reply:
Sorry, they had been at a point… I reverted. 
Should I keep the commas at the end ?